packages/auth-sdk/user.tsx

```diff
@@ -22,7 +22,7 @@ export function useAuthenticatedUser() {
       userKeys.user,
       () => fetchAuthenticatedUser(appContext.ifixitOrigin),
       {
-         retry: false,
+         retryOnMount: false,
          staleTime: Infinity,
       }
    );
```

Comment:
I feel like it might be more intuitive to pass `useQuery` options into the hook, as opposed to hardcoding them. As a caller of `useAuthenticatedUser`, it's not very explicit that `retryOnMount` and `staleTime` are anything different than their defaults. This is pretty minor since we only have one use case, but I'm just curious if there's a reason we _shouldn't_ pass these options into the hook.

Reply:
I think these options should just be considered an implementation details of the `useAuthenticatedUser` hook. Letting the consumer change those would affect all the other consumers, since the behaviour is shared with React Query.

Reply:
👍🏻 

> Letting the consumer change those would affect all the other consumers

Didn't consider this